src/main/java/uk/gov/hmcts/darts/common/helper/SystemUserHelper.java

```diff
@@ -20,8 +24,18 @@ public class SystemUserHelper {
 
     private Map<String, String> systemUserGuidMap;
 
+    private final UserAccountRepository userAccountRepository;
+
     public String findSystemUserGuid(String systemUserName) {
         return systemUserGuidMap.get(systemUserName);
     }
 
+    public UserAccountEntity getSystemUser() {
+        UserAccountEntity user = userAccountRepository.findSystemUser(findSystemUserGuid("housekeeping"));
```

Comment:
do we just need its reference?
maybe method should be called `getHousekeepingUserRef`?

If not, can we cache the result so that we don't have to keep looking it up?

Reply:
I think ref is a good idea. accept.

Reply:
ACCEPT.